sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java

```diff
@@ -41,6 +41,7 @@
   public static final String LEAK_PERIOD_MODE_PREVIOUS_VERSION = "previous_version";
 
   private static final String DEFAULT_LEAK_PERIOD = LEAK_PERIOD_MODE_PREVIOUS_VERSION;
+  private static final String DEFAULT_LONG_LIVED_BRANCHES_REGEX = "feature/.*";
```

Comment:
I think feature branches are typically short-lived, so this is not a good default pattern for long-lived branches. I think that for example `(branch|release)-.*` would be better. We could look at some open-source projects that manage multiple branches to support multiple recent releases and see if there's a common pattern.